torch/csrc/distributed/c10d/reducer.h

```diff
@@ -13,6 +13,10 @@
 
 namespace c10d {
 
+constexpr int DEFAULT_FIRST_BUCKET_BYTES = int(1024 * 1024);
+constexpr int DEFAULT_BUCKET_BYTES_CAP = int(25 * 1024 * 1024);
+constexpr int DEFAULT_BROADCAST_BUCKET_BYTES = int(250 * 1024 * 1024);
```

Comment:
Let's stay consistent with other parts of our code base and use `kDefaultBroadcastBucketBytes` instead.